TestAssets/TestProjects/PortableTestApp/Program.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace PortableApp
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
```

Comment:
oversight, will fix while working on parsing the output. 1 passing, 1 failing

---
In reply to: [66346482](https://github.com/dotnet/core-setup/pull/97#discussion_r66346482) [](ancestors = 66346482)